docs/coding.rst

```diff
@@ -654,12 +676,12 @@ Picto
     Example: ``l10n_image=True``
 
 
-- lazy_loading
-    Boolean to provide “lazy” value for “loading” attribute. This will be “eager” by default. Lazy loading defers fetching of images to a browser decision based on user scroll and connection.
+- loading
+    String to provide value for image loading attribute. This will use browser default ("eager") if not set. Lazy loading defers fetching of images to a browser decision based on user scroll and connection.
 
-    Default: False
+    Default: None
```

Comment:
Two things shifted this in my mind from default "eager" to default "None"
1. "eager" is already the spec default when the loading attribute is not set, so it felt redundant to add the attribute in these cases
2. Chrome does seem to allow [this "auto" value](https://web.dev/browser-level-image-lazy-loading/#the-loading-attribute) to determine when images should load when the loading attribute is not set, which would be overridden by an "eager" default in our macro. I like the idea that no attribute means "let the browser handle it", and in Chrome's case that means "auto" instead of "eager"